documentation/wiki/pages/Polymer Editor - Open file.md

```diff
@@ -5,9 +5,10 @@
 
 #### 1. Context
 
+Supported formats are: *.mol (v3000), *.HELM, *.txt , .*doc, .docx, .rtf
```

Comment:
Text files are only supported if they have MOL or HELM notation inside, we should treat text files with other content as unsupported.